component-metadata-schema.json

```diff
@@ -1,261 +0,0 @@
-{
```

Comment:
Are you deleting this file intentionally? https://github.com/dapr/components-contrib/actions/runs/3694999058/jobs/6256860667 also seems to be failing due to this.